brave/brave-core
# Sign hardware transaction in wallet panel

Resolves https://github.com/brave/brave-browser/issues/17971

- Proxy hardware transactions signing to device
- Do not calculate keccak for Ledger transactions 
- Added hardware information to AccountInfo

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Connect device to Ganache
- Send money from Ganache account to device
- Send money back from Device to Ganache, pay attention to transaction fee
- Check if all transactions successful
